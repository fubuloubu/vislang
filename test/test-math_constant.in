1.000
2.000
3.765
