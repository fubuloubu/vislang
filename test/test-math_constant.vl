<vl:BLOCK name="plus_constant">
    <vl:INPUT name="in" datatype="single" scope="local" size="1"/>
    <vl:CONSTANT name="some_val" datatype="single" size="1" value="1.235"/>
    <vl:SUM name="summer" datatype="single">
        <vl:CONNECTION to="input1" from="in"/>
        <vl:CONNECTION to="input2" from="some_val"/>
    </vl:SUM>
    <vl:OUTPUT name="out" datatype="single" scope="local" size="1">
        <vl:CONNECTION to="out" from="summer"/>
    </vl:OUTPUT>
</vl:BLOCK>
