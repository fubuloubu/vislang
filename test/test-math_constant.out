2.235
3.235
5.000
